tests/pytests/data/MVA_2B2_01_02329N002E0302/SEL_M_V01.TSC

```diff
@@ -11,7 +11,7 @@ SCLK_KERNEL_ID                = ( @2009-06-10T12:57:27.4670 )
 SCLK_DATA_TYPE_131       = ( 1 )
 SCLK01_N_FIELDS_131      = ( 1 )
 SCLK01_MODULI_131        = ( 4294967296 )
-SCLK01_OFFSETS_131       = ( 0 0 )
+SCLK01_OFFSETS_131       = ( 0 )
```

Comment:
According to the [SCLK Required Reading](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/req/sclk.html#Format%20constant%20assignments), this is now correct. There should be 1 offset per field and `SCLK01_N_FIELDS_131      = ( 1 )` only indicates 1 field.